JSON_for_IO-Link.yaml

```diff
@@ -3744,8 +3778,6 @@ components:
         - vendorId
         - deviceId
         - ioLinkRevision
-        - vendorName
```

Comment:
It might be a problem to remove a previously required property. On the other hand, I can understand that there are devices which do not have these parameter since they are not mandatory.

Reply:
I agree, don't know why this is removed here from **deviceIdentificationGet**. I believe it was because of a linter error.

Reply:
Fixed this issue